tvtk/tests/test_tvtk.py

```diff
@@ -636,6 +657,180 @@ def test_null_string_wrapper(self):
             ('caption', 'GetCaption') in cap._updateable_traits_, True)
         self.assertEqual('caption' in cap._full_traitnames_list_, True)
 
+    def test_spider_plot_actor(self):
+        """ Test SpiderPlotActor Get/SetAxisLabel works
+        """
+        actor = tvtk.SpiderPlotActor()
```

Comment:
It would be nice if there were a comment as to why this object is tested that would be helpful.